io/disk/Avago_storage_adapter/avago9361_vd.py

```diff
@@ -99,16 +108,72 @@ def test_maxvd(self):
                     self.vd_details()
                 self.vd_delete()
 
+    def test_init(self):
+
+        """
+        Test case to start Fast and Full init on VD
+        """
+        self.vd_create('WB', 'ra', 'cached', 256)
```

Comment:
Not required, as creating the VD here is for different purpose. And I anyway am testing create VD with details earlier.